MobiusCore/Source/EffectHandlers/EffectHandler.swift

```diff
@@ -17,16 +17,23 @@
 // specific language governing permissions and limitations
 // under the License.
 
-// Primitive EffectHandler
-public struct EffectHandler<Effect, Event> {
-    public let handle: (Effect, @escaping Consumer<Event>) -> Void
-    public let disposable: Disposable
+public protocol EffectHandler {
+    associatedtype Input
+    associatedtype Output
+
+    func handle(_ input: Input, _ output: @escaping Consumer<Output>) -> Disposable
+}
+
+public struct AnyEffectHandler<Input, Output>: EffectHandler {
+    private let handler: (Input, @escaping (Output) -> Void) -> Disposable
```

Comment:
What is the rationale for not using the `Consumer` type alias here? It seems like using it would make the code a little clearer, because you get a hint about what the closure is for.

Reply:
(The same comment applies to other places in this file)

Reply:
No reason, I just forgot